packages/react-button/src/components/Button/useButtonStyles.ts

```diff
@@ -248,9 +248,10 @@ const useRootFocusStyles = makeStyles({
       ${theme.alias.shadow.shadow4},
       0 0 0 2px ${theme.alias.color.neutral.strokeFocus2}
     `,
+    zIndex: 1,
   })),
   circular: createFocusIndicatorStyleRule(theme => ({
-    borderRadius: theme.global.borderRadius.circular,
+    borderRadius: '50000px',
```

Comment:
If you always want your border radii to match between the control and the focus indicator, you could extract to a constant.

Reply:
I'll revert this to using the theme token since we are going to be changing that anyway as stated in my comment above.